### PR TITLE
extract facilities as new field

### PR DESCRIPTION
**Pull request by krisbukovi**
addresses issue #25, facilities must be in the format shown below to be extracted:

```<named-content content-type="facility" xlink:href="FacilityName">```
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;```FacilityName```
```</named-content>```

### CODE REVIEW

File: tests/test_unit/stub_data/test.xml [hunk 1]
```diff
@@ -126,6 +126,7 @@
       <ack>
          <title>Acknowledgments</title>
          <p>WE ACKNOWLEDGE.</p></ack>
+      <p><italic>Facility:</italic> <named-content content-type="facility" xlink:href="FacilityName"><italic>FacilityName</italic></named-content>.</p>
```
Comment (marblestation) [inline tests/test_unit/stub_data/test.xml:169]: We don't have a unit test to verify this gets extracted, right?

**Review by marblestation — CHANGES_REQUESTED** [review]
The code extracts the facilities and stores it to the metadata file on disk but it never sends the data to the master pipeline via RabbitMQ, so we won't populate the facility field as the issue was requesting. We need to update https://github.com/adsabs/ADSPipelineMsg/blob/master/specs/fulltext_update.proto to include the facility field, make a new release, update the requirements in this package, and make sure it send the information to master pipeline here: https://github.com/adsabs/ADSfulltext/blob/767bbd8cdf74a50098de5d01a07416af3c6d3c01/adsft/tasks.py#L118

Then, we'll need to check master pipeline to make sure that it grabs this content and populates the corresponding field that is finally sent to solr.

File: tests/test_unit/stub_data/test.xml (hunk 1 above)
Comment (krisbukovi) [reply tests/test_unit/stub_data/test.xml:169]: Unit test added

**Comment by krisbukovi** [comment]
As we discussed, the code does send the data to the master pipeline via RabbitMQ so no change has been made in regards to that. The requirements for ADSPipelineUtils has been updated to most recent release (v.1.0.9).

**Review by marblestation — CHANGES_REQUESTED** [review]
Unfortunately this will not send the data to master pipeline because ADSPipelineUtils v1.0.9 requires ADSPipelineMsg v1.0.8 and not v1.0.9, but it's in v1.0.9 where you added the facility field. Fortunately, the change is quick and easy: we need to update ADSPipelineUtils requirements, create a new version (v1.0.10) and require that new version in here. Also, this PR conflicts with the current code, we need to rebase it.

**Comment by krisbukovi** [comment]
Okay, ADSfulltext is pointing to the most recent release of ADSPipelineUtils v1.0.15 which is pointing to the most recent release of ADSPipelineMsg v1.0.10.

**Review by marblestation — APPROVED** [review]
Awesome! Thanks! This is ready to be merged now :+1:

**Pull request merged**